bin/aurelia.js

```diff
@@ -56,17 +56,15 @@ function handleCommands(env) {
   }
 
   aurelia.init({
-    env : env
+      env : env
+    , argv: argv
   });
 
-  if (env.configPath) {
+  if (env.configPath && env.modulePath) {
     require(env.configPath);
     logger.info('Using Aureliafile: %s', env.configPath);
   } else {
     logger.log('warn', 'Aureliafile not found');
   }
-
-  process.nextTick(function() {
-    aurelia.run(process.argv);
-  });
+  aurelia.run(argv);
```

Comment:
@joelcoxokc  `process.nextTick()` make sure the `Aurelia` object is instantiated properly before `aurelia.run` executes.